.readthedocs.yml

```diff
@@ -16,8 +16,15 @@ sphinx:
 formats:
   - htmlzip
 
+submodules:
+  include: all
+  recursive: true
+
 ## Optionally set the version of Python and requirements required to build your docs
 python:
   version: 3.7
   install:
+    - requirements: requirements-dev.txt
```

Comment:
I'm really uncomfortable with this. Building documentation shouldn't require installing the whole development dependency set. The reason you're doing this is because you're directly importing the package in the doc building step - which is antipattern. We should separate the responsibilities here

Reply:
I'm just messing around with readthedocs right now.

Reply:
Importantly, I was poking around to see if I could get readthedocs to import WhyLogs so we could run the example scripts to generate example documentation.  But I can't install protoc without more effort, so I'm scrapping that for now.